engine/apps/api/views/features.py

```diff
@@ -58,4 +59,15 @@ def _get_enabled_features(self, request):
             if request.auth.organization.pk in enabled_web_schedules_orgs.json_value["org_ids"]:
                 enabled_features.append(FEATURE_WEB_SCHEDULES)
 
+        enabled_webhooks2_orgs = DynamicSetting.objects.get_or_create(
```

Comment:
FYI, there is a `is_webhooks_enabled_for_organization` helper in webhooks/utils.py that I think you could use here.